cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -176,51 +223,55 @@ var Plugin = new Class({
                 that.options.plugin_name,
                 that._getPluginBreadcrumbs()
             );
-        });
+        };
 
-        // adds edit tooltip
-        this.ui.container.on(this.pointerOverAndOut + ' ' + this.touchStart, function (e) {
-            // required for both, click and touch
-            // otherwise propagation won't work to the nested plugin
-            e.stopPropagation();
-            if (e.type === 'touchstart') {
-                CMS.API.Tooltip._forceTouchOnce();
-            }
-            var name = that.options.plugin_name;
-            var id = that.options.plugin_id;
+        this.ui.dragitem.on(this.doubleClick, dblClickHandler);
 
-            var placeholderId = that._getId(that.ui.dragitem.closest('.cms-dragarea'));
-            var placeholder = $('.cms-placeholder-' + placeholderId);
+        if (!Plugin._isContainingMultiplePlugins(this.ui.container)) {
+            this.ui.container.on(this.doubleClick, dblClickHandler);
+            this.ui.container.on(this.pointerOverAndOut + ' ' + this.touchStart, function (e) {
+                // required for both, click and touch
+                // otherwise propagation won't work to the nested plugin
+                e.stopPropagation();
+                if (e.type === 'touchstart') {
+                    CMS.API.Tooltip._forceTouchOnce();
```

Comment:
yes we do need to call it, since it won't propagate and won't be catched by the tooltip itself, and no, we shouldn't make it public